aea/configurations/constants.py

```diff
@@ -22,12 +22,14 @@
 from aea.configurations.base import DEFAULT_LICENSE as DL
 from aea.configurations.base import DEFAULT_REGISTRY_PATH as DRP
 from aea.configurations.base import PublicId
-from aea.crypto.fetchai import FetchAICrypto
+from aea.crypto.cosmos import CosmosCrypto
+from aea.crypto.helpers import COSMOS_PRIVATE_KEY_FILE
 
 DEFAULT_CONNECTION = PublicId.from_str("fetchai/stub:0.6.0")
 DEFAULT_PROTOCOL = PublicId.from_str("fetchai/default:0.3.0")
 DEFAULT_SKILL = PublicId.from_str("fetchai/error:0.3.0")
-DEFAULT_LEDGER = FetchAICrypto.identifier
+DEFAULT_LEDGER = CosmosCrypto.identifier
```

Comment:
Just a thought: needless to say, this is an important change for the framework. Historically, we had first `fetchai` before `cosmos`, both in code and in docs.

Of course, in code doesn't matter much (apart from hard-coded strings that might be missed), it should work anyway. But in the examples, tutorials etc. we often introduce first `fetchai` and then `cosmos`, if any. Have the docs been reviewed with that in mind?

(I might see it by myself, but I ask to the opener of the PR since it is quite big.)

Reply:
I did change the docs; but might have missed occasional changes.